src/lib/TranscriptEditor/MediaPlayer/PlayerControls.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './PlayerControls.module.css';
+class PlayerControls extends React.Component {
+    constructor(props) {
+        super(props);
+    }
+    // to handle backward and forward mouse pressed on btn
+    // set a 300 ms  interval to repeat the 
+    // backward or forward function
+    // on mouseUp the interval is cleared
+    setIntervalHelperBackward =() => {
+       this.interval =  setInterval(() => {
+                this.props.skipBackward();
+        }, 300);       
+    }
+
+    setIntervalHelperForward =() => {
+        this.interval =  setInterval(() => {
+                 this.props.skipForward();
+         }, 300);       
+     }
+
+    clearIntervalHelper = () => {
+        clearInterval( this.interval);
+    }
+
+    setTimecodeOffSetHelper = () => {
+        this.props.setTimeCodeOffset( prompt('Add a timecode offset hh:mm:ss:ff'))
+    }
+
+    render() {
+        return (
+          <div>
+            {/* Play Btn  */}
```

Comment:
Instead of these comments, let's put classNames for each button. Self-describing and useful down the line too 👍

Reply:
ok removing these comments for now, waiting on the CSS classes for when we address the UI.